### PR TITLE
feat: add select/multi-select components

### PR DESCRIPTION
**Pull request by jrasm91**
- Add the `Select` component
- Add the `MultiSelect` component
- Add a color option, `muted` to the `Text` component
- Add a `npm run create MyNewComponent` script
- Change the default border to match the card and app-shell

### CODE REVIEW

File: scripts/create.js
Comment (danieldietzler) [inline scripts/create.js:1]: That is pretty neat.

File: src/lib/internal/Select.svelte [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+		if (ref && contentRef) {
+			contentRef.style.width = `${ref.clientWidth}px`;
+		}
+	});
+
+	const findOption = (value: string) => options.find((option) => option.value === value);
+
+	const onValueChange = (items: string[] | string) => {
+		values = multiple
+			? ((items as string[]).map(findOption) as T[])
```
Comment (danieldietzler) [inline src/lib/internal/Select.svelte:89]: Why do you not need the .filter here? Won't you end up with an array that also has `undefined` entries?

File: src/lib/internal/Select.svelte [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+			<Select.Content
+				bind:ref={contentRef}
+				class="max-h-96 select-none rounded-xl border bg-light py-3 text-dark outline-none"
+				sideOffset={10}
+			>
+				<Select.ScrollUpButton class="flex w-full items-center justify-center">
+					<Icon icon={mdiArrowUp}></Icon>
+				</Select.ScrollUpButton>
+				<Select.Viewport>
+					{#each options as { value, label, disabled }, i (i + value)}
```
Comment (danieldietzler) [inline src/lib/internal/Select.svelte:143]: What's the reason we key on `i + value`?

Comment (jrasm91) [reply src/lib/internal/Select.svelte:143]: Value doesn't have to be unique

**Review by danieldietzler — APPROVED** [review]

**Pull request merged**